### ISSUE TITLE
electron-builder fails to package the electron app on Ubuntu

### ISSUE THREAD

**Issue by epezhman** [octalmage/robotjs#246]
I'm trying to use Robot.js in my electron app, I got the app running with `electron-prebuilt` by navigating to `node-modules/robotjs` and running 
`sudo HOME=~/.electron-gyp node-gyp rebuild --target=1.3.4 --arch=x64 --dist-url=https://atom.io/download/atom-sh`

but when I'm trying to build the app by` electron-builder` I get this: any ideas? Thanks

```
Error output:
prebuild ERR! clean error 
prebuild ERR! stack Error: EACCES: permission denied, unlink 'build/Makefile'
prebuild ERR! stack     at Error (native)
prebuild ERR! not ok 
prebuild ERR! build Error: EACCES: permission denied, unlink 'build/Makefile'
prebuild ERR! build     at Error (native)

npm ERR! Linux 4.4.0-45-generic
npm ERR! argv "/usr/bin/nodejs" "/usr/bin/npm" "rebuild" "--production" "--build-from-source" "--cache-min" "999999999"
npm ERR! node v4.6.1
npm ERR! npm  v3.10.9
npm ERR! code ELIFECYCLE
npm ERR! robotjs@0.4.5 install: `prebuild --install`
npm ERR! Exit status 2
npm ERR! 
npm ERR! Failed at the robotjs@0.4.5 install script 'prebuild --install'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the robotjs package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     prebuild --install
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs robotjs
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls robotjs
npm ERR! There is likely additional logging output above.
```

## Your Environment
* RobotJS version: ^0.4.5
* Electron version: 1.4.3
* npm version: 3.10.9
* Operating System: Ubuntu 16.04

**Comment by twang2218** [comment]
```
Error: EACCES: permission denied, unlink 'build/Makefile'
```

It's permission error, where is your code located? and what's permission of the `build/Makefile` is?

**Comment by octalmage** [comment]
This should be resolved by v0.4.6, please let me know if you still encounter this and we can reopen the issue.

**Comment by matheo** [comment]
@octalmage what's the issue here? I'm getting the same error :(
